Testscripts/Windows/KDUMP-TestScript.ps1

```diff
@@ -115,14 +151,16 @@ function Main {
         Write-LogWarn "Distro is not supported or kernel config does not allow auto"
         return "SKIPPED"
     }
+
+    $waitToStartTime=900
+
     # Rebooting the VM in order to apply the kdump settings
     Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort `
         -command "sync; reboot" -runAsSudo -RunInBackGround | Out-Null
     Write-LogInfo "Rebooting VM $VMName after kdump configuration..."
-    Start-Sleep -Seconds 10 # Wait for kvp & ssh services stop
 
     # Wait for VM boot up and update ip address
-    Wait-ForVMToStartSSH -Ipv4addr $Ipv4 -StepTimeout 360 | Out-Null
+    Wait-ForVMToStartSSH -Ipv4addr $Ipv4 -StepTimeout $waitToStartTime | Out-Null
```

Comment:
after extending the time, not enough?

Reply:
No. Again I added a logic to make sure VM booted up without waiting for fixed time.

Reply:
does the 22 open but VM still can't be login?

Reply:
Yes

Reply:
I have a test, we have known bug when trigger crash on some specified CPU range, in this case, we have to wait till timeout.